README.md

```diff
@@ -120,9 +120,7 @@ You can cache the installed gems with these two steps:
         restore-keys: |
           bundle-use-ruby-${{ matrix.os }}-${{ matrix.ruby }}-
     - name: bundle install
-      run: |
-        bundle config path vendor/bundle
-        bundle install --jobs 4 --retry 3
+      run: bundle install --deployment --jobs 4
```

Comment:
On Bundler 2 this triggers a warning:

$ bundle install --deployment
[DEPRECATED] The `--deployment` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set deployment 'true'`, and stop using this flag


Could you change it accordingly?

Reply:
I'd suggest the shorter form `bundle config deployment true`.
That's global, but probably good in CI?
`bundle config --local deployment true` seems fine too.